gpt4all-chat/qml/ChatDrawer.qml

```diff
@@ -263,91 +250,47 @@ Drawer {
             }
         }
 
-        Button {
+        MyButton {
             id: checkForUpdatesButton
             anchors.left: parent.left
             anchors.right: parent.right
             anchors.bottom: downloadButton.top
             anchors.bottomMargin: 10
-            padding: 15
-            contentItem: Text {
-                text: qsTr("Updates")
-                horizontalAlignment: Text.AlignHCenter
-                color: theme.textColor
-
-                Accessible.role: Accessible.Button
-                Accessible.name: text
-                Accessible.description: qsTr("Use this to launch an external application that will check for updates to the installer")
-            }
-
-            background: Rectangle {
-                opacity: .5
-                border.color: theme.backgroundLightest
-                border.width: 1
-                radius: 10
-                color: theme.backgroundLight
-            }
-
+            text: qsTr("Updates")
+            Accessible.role: Accessible.Button
+            Accessible.name: text
+            Accessible.description: qsTr("Use this to launch an external application that will check for updates to the installer")
             onClicked: {
                 if (!LLM.checkForUpdates())
                     checkForUpdatesError.open()
             }
         }
 
-        Button {
+        MyButton {
             id: downloadButton
             anchors.left: parent.left
             anchors.right: parent.right
             anchors.bottom: aboutButton.top
             anchors.bottomMargin: 10
-            padding: 15
-            contentItem: Text {
-                text: qsTr("Downloads")
-                horizontalAlignment: Text.AlignHCenter
-                color: theme.textColor
-
-                Accessible.role: Accessible.Button
-                Accessible.name: text
-                Accessible.description: qsTr("Use this to launch a dialog to download new models")
-            }
-
-            background: Rectangle {
-                opacity: .5
-                border.color: theme.backgroundLightest
-                border.width: 1
-                radius: 10
-                color: theme.backgroundLight
-            }
-
+            text: qsTr("Downloads")
+            Accessible.role: Accessible.Button
+            Accessible.name: text
+            Accessible.description: qsTr("Use this to launch a dialog to download new models")
             onClicked: {
                 downloadClicked()
             }
         }
 
-        Button {
+        MyButton {
             id: aboutButton
             anchors.left: parent.left
             anchors.right: parent.right
             anchors.bottom: parent.bottom
             padding: 15
```

Comment:
You didn't remove the padding here...

Reply:
Oh.. my bad - will fix this, rebase and repush